web-common/src/features/charts/render/vega-config.ts

```diff
@@ -16,9 +16,6 @@ export const getRillTheme: () => Config = () => ({
   autosize: {
     type: "fit-x",
   },
-  mark: {
-    tooltip: true,
-  },
```

Comment:
What's the idea behind removing this? We also use the same config for our custom dashboard charts so this removes the default tooltips from those charts.

Reply:
> What's the idea behind removing this?

Keeping this in the global config will trigger an empty tooltip on the brush interval.

```
  mark: {
    tooltip: true,
  },
```


https://github.com/user-attachments/assets/cdda169d-bd64-44a7-bb9a-b55070425edf

Reply:
If the global tooltip config is absolutely needed, I can enable the tooltip flag for customDashboard.

Reply:
https://github.com/rilldata/rill/pull/5483/commits/86805de59ca62a3ccfeb317ad52e8eac113dec4b